### DISCUSSION TITLE
Kerberos: approach to login with a different user account

### DISCUSSION THREAD

**Discussion by oculos** [keycloak/keycloak#30281] [Q&A]
I have testet Keycloak with kerberos, and I think it works really well.

However, the biggest issue is that a user never manages to log off. In our case, some users have two accounts, and it would be nice for them to be able to login with the other.

Is there a way to have a a logout redirect to a username/password window, to give the user the opportunity to log in with another account?

I see that FreeIPA does it beautifully: logging out redirects the user to the username/password. If you reload that page, it will log you in with Kerberos automatically.

**Comment by nhart12** [comment]
I'm also looking for this similar behavior. It would be cool if we could pass prompt=login after a logout but it appears that isn't respected
https://github.com/keycloak/keycloak/issues/8989
